db/data/cardGamesData.js

```diff
@@ -0,0 +1,139 @@
+let cardGamesData = [
```

Comment:
Thanks for making sure to add the content. I know this takes time and really appreciate the hard work!